.github/workflows/approved-prs.yml

```diff
@@ -0,0 +1,38 @@
+name: "Prompt reviewers to merge PRs on behalf of authors"
+
+permissions:
+  contents: read
+
+on:
+  pull_request_review:
+    types:
+      - submitted
+
+jobs:
+  merge_on_behalf_information_comment:
```

Comment:
I think the convention is to use dashes instead of underscores here.

Reply:
Fixed.